### PR TITLE
Diagram layout improvements

### LINKED ISSUE #29: Improve diagram CSS of containment diamonds
The diamonds are not filled black but should be:
image

### LINKED ISSUE #30: Improve auto layouting
The auto-layouted diagram elements heavily overlap and the labels often are overlapped by elements. Often they are also upside down.

### PR DESCRIPTION
**Pull request by planger**
Fix #29 and fix #30

### CODE REVIEW

**Comment by eneufeld** [comment]
LGTM

File: web/docu-app/packages/sprotty-component/package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "snabbdom": "^0.7.0",
     "snabbdom-jsx": "^0.4.2",
     "snabbdom-virtualize": "^0.7.0",
-    "sprotty": "next"
+    "sprotty": "next",
+    "sprotty-elk": "^0.5.0-next.cd1ab08"
```
[the remark this answers is not in the document]
Comment (planger) [reply web/docu-app/packages/sprotty-component/package.json]: Good point, should we use "next"? I've proposed a fix for an issue with edge labels, which aren't layouted by the default `ElkLayoutEngine` (see https://github.com/eclipse/sprotty-layout/issues/3). Once this is merged and if we use "next", we could get rid of our spezialized layout engine.

File: web/docu-app/packages/sprotty-component/typings/elkjs/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
+declare module "elkjs/lib/elk-api" {
```
[the remark this answers is not in the document]
Comment (planger) [reply web/docu-app/packages/sprotty-component/typings/elkjs/index.d.ts]: I copied the typings from sprotty-layout for now. But I assume we could also import them from sprotty-layout.

**Comment by planger** [comment]
> So this enabled client side layout ?

Exactly, I figured, now when we have a browser back-end we might as well just run the layouting here to take advantage of the knowledge about the actual rendered element sizes.

> Does it still make sense to keep the server side layout?

I think we can remove it from the dynamic docu but keep it in the static version.

**Comment by planger** [comment]
Yes, I created #32 and #33. Thanks!

**Comment by planger** [comment]
And #34

**Pull request merged**